content/chat.adoc

```diff
@@ -36,16 +36,14 @@ https://libera.chat/guides/clients[Find an IRC client].
 
 General discussions channel for Jenkins users and developers.
 It used to be the primary one until non-IRC communication options were introduced.
-link:/project/board[Board members and officers] typically have _op_ (@) in this channel.
-Established contributors typically have _voice_ (+) in this channel.
 
 **Tip:** If you have a question about Jenkins, just ask it. Don't ask whether you can ask a question.
 It's common to not receive an answer for quite a while if no active contributors are currently checking IRC.
 In that case, just stick around, and periodically (every few hours) ask your question again.
 
 === `#jenkins-infra`
 
-Discussions of the link:/projects/infrastructure/[Jenkins project infrastructure], i.e. most services running on `jenkins.io`, `jenkins-ci.org`, and related domains.
+Discussions of the link:/projects/infrastructure/[Jenkins project infrastructure], i.e. most services running on `jenkins.io`, `ci.jenkins.io`, and related domains.
```

Comment:
I believe that the phrasing is intentional here.  The artifact repository is at https://repo.jenkins-ci.org and needs to remain at that location so that it does not break Maven builds.  The items in the list are domain name suffixes that we own, not example domain names.
```suggestion
Discussions of the link:/projects/infrastructure/[Jenkins project infrastructure], i.e. most services running on `jenkins.io`, `jenkins-ci.org`, and related domains.
```

Reply:
Good point, https://github.com/jenkins-infra/jenkins.io/pull/6031